### PR TITLE
Panels for a tumor purity figure, part 1

### PR DESCRIPTION
**Pull request by sjspielman**
Towards #1665 

This PR updates three modules, including code and docs, to export plots that can be used towards the tumor purity SI figure. As proposed in #1665, these are (with nested bullets indicating relevant changed files) - 

- **Panel A** distribution of tumor purity by cancer group: `analyses/tumor-purity-exploration/plots/cancer_group_tumor_fraction.pdf`
  - `analyses/tumor-purity-exploration/01_explore-tumor-purity.Rmd` and html
  - `analyses/tumor-purity-exploration/README.md`

- **Panel D** remake UMAP with high purity tumors only: `analyses/transcriptomic-dimension-reduction/plots/umap_tumor-purity-threshold.pdf`
  - `analyses/transcriptomic-dimension-reduction/analyses/transcriptomic-dimension-reduction/06-umap-tumor-purity-threshold.Rmd` and html
  - `analyses/transcriptomic-dimension-reduction/README.md`

- **Panel E** quanTIseq estimations of immune fractions across cancer groups: `analyses/immune-deconv/plots/tumor-purity-threshold_quantiseq-cancer-groups.pdf`
  - `analyses/immune-deconv/03-quantiseq-at-threshold.Rmd` and html
  - `analyses/immune-deconv/README.md`

### CODE REVIEW

File: analyses/tumor-purity-exploration/01_explore-tumor-purity.Rmd [hunk 1]
```diff
@@ -125,16 +139,33 @@ tumor_palette_df <- metadata_tumor_wgs %>%
...
 # Broad histology
-plot_tumor_violins(tumor_palette_df, broad_histology_display, broad_histology_hex, "Broad histology")
+broad_histology_violions <- plot_tumor_violins(tumor_palette_df, broad_histology_display, broad_histology_hex, "Broad histology")
+```
+
+
+```{r, fig.width = 10, fig.height = 4}
+# Print plots
+cancer_group_violins
+broad_histology_violions
```
Comment (jaclyn-taroni) [inline analyses/tumor-purity-exploration/01_explore-tumor-purity.Rmd:152]:
```suggestion
broad_histology_violins
```

File: analyses/tumor-purity-exploration/01_explore-tumor-purity.Rmd [hunk 2]
```diff
@@ -125,16 +139,33 @@ tumor_palette_df <- metadata_tumor_wgs %>%
   drop_na()
 
 # Cancer group
-plot_tumor_violins(tumor_palette_df, cancer_group_display, cancer_group_hex, "Cancer group")
+cancer_group_violins <- plot_tumor_violins(tumor_palette_df, cancer_group_display, cancer_group_hex, "Cancer group")
 
 # Broad histology
-plot_tumor_violins(tumor_palette_df, broad_histology_display, broad_histology_hex, "Broad histology")
+broad_histology_violions <- plot_tumor_violins(tumor_palette_df, broad_histology_display, broad_histology_hex, "Broad histology")
```
Comment (jaclyn-taroni) [inline analyses/tumor-purity-exploration/01_explore-tumor-purity.Rmd:145]:
```suggestion
broad_histology_violins <- plot_tumor_violins(tumor_palette_df, broad_histology_display, broad_histology_hex, "Broad histology")
```

**Review by jaclyn-taroni — APPROVED** [review]
Found typos of no functional consequence, but other than that LGTM 👍🏻

**Comment by jharenza** [comment]
I think it would be nice to add Panel A to the manuscript, and the remaining panels (along with A as the multi-panel figure @jaclyn-taroni suggested) to the reviewer response. Can we add this as Supp Figure 3A, perhaps splitting the space with the oncoprint, making that B and so on?

**Pull request merged**